testing/src/test/java/io/stargate/it/BaseOsgiIntegrationTest.java

```diff
@@ -73,15 +78,43 @@
     unFinal();
   }
 
-  static Starter stargateStarter;
-  public static String stargateHost = "127.0.0.11";
   public boolean enableAuth;
+
   protected final ClusterConnectionInfo backend;
 
+  public static List<Starter> stargateStarters = new ArrayList<>();
+  private static List<String> stargateHosts = new ArrayList<>();
+  public static final Integer numberOfStargateNodes = 3;
+
+  static {
+    for (int i = 1; i <= numberOfStargateNodes; i++) {
+      stargateHosts.add("127.0.0.1" + i);
+    }
+  }
+
   public BaseOsgiIntegrationTest(ClusterConnectionInfo backend) {
     this.backend = backend;
   }
 
+  public static String getStargateHost() {
+    return getStargateHost(0);
+  }
+
+  public static String getStargateHost(int stargateInstanceNumber) {
+    return stargateHosts.get(stargateInstanceNumber);
+  }
+
+  public static List<String> getStargateHosts() {
+    return stargateHosts;
+  }
+
+  public static List<InetAddress> getStargateInetSocketAddresses() throws UnknownHostException {
+    return Arrays.asList(
```

Comment:
Should we build this list dynamically using `numberOfStargateNodes` ? That way we don't have to worry about this getting out of sync if that number is changed.